app/config/Seeds/ArticlesSeed.php

```diff
@@ -0,0 +1,47 @@
+<?php
+declare(strict_types=1);
+
+use Migrations\AbstractSeed;
+
+/**
+ * Articles seed.
+ */
+class ArticlesSeed extends AbstractSeed
+{
+    /**
+     * Run Method.
+     *
+     * Write your database seeder using this method.
+     *
+     * More information on writing seeds is available here:
+     * https://book.cakephp.org/phinx/0/en/seeding.html
+     *
+     * @return void
+     */
+    public function run(): void
+    {
+        $data = [
+            [
+                'user_id' => 1,
+                'title' => 'First Post',
+                'slug' => 'first-post',
+                'body' => 'This is the first post.',
+                'published' => 1,
+                'created' => date('Y-m-d H:i:s'),
+                'modified' => date('Y-m-d H:i:s'),
+            ],
+        ];
+
+        $table = $this->table('articles');
+
+        foreach ($data as $articleData) {
+            $existingArticle = $this->query("SELECT * FROM articles WHERE slug = :slug", ['slug' => $articleData['slug']])->fetch();
```

Comment:
In this PR, you've learn 2 ways to retrieve the records for migrations seed:
1. query()->fetch()
2. fetchRow()

Both will retrieve the first record only. Since the articles' slug and users' email is unique, then it's still safe. 

If we've to query a field that is not guaranteed unique and want to retrieve 1st row only, then it's better to add SQL statement `LIMIT 1` since fetch still retrieve all records and return the first record only.

Notes: 
1. In actual project, please standardize the usage either to use `query()->fetch()` & `fetchRow()`. Code uniformity & consistency is important.
2. This comment doesn't need to be fix. It's OK for now.